ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/sno/cucushift-installer-rehearse-gcp-ipi-sno-workflow.yaml

```diff
@@ -0,0 +1,19 @@
+workflow:
+  as: cucushift-installer-rehearse-gcp-ipi-sno
+  steps: 
+    pre:
+    - chain: cucushift-installer-rehearse-gcp-ipi-provision
+    post:
+    - chain: cucushift-installer-rehearse-gcp-ipi-deprovision
+    env:
+      CONTROL_PLANE_REPLICAS: "1"
```

Comment:
suggest to introduce a dedicate provision chain to use `single-node-conf-gcp` step here.

Reply:
I checked the chains `azure-ipi-sno-provision` and `aws-ipi-disconnected-private-sno-provision`, they do include the step `single-node-conf-{azure,aws}`, and I guess they were doing so because their `ipi-conf-{azure,aws}` steps don't support the env var `CONTROL_PLANE_REPLICAS`. 

As the `ipi-conf-gcp` step does support the env var, using this env var directly in the workflow can be without the dedicated provision chain. Do you still want me adding the chain? Thanks!

Reply:
Yeah, I suggest to keep the same style as aws and azure.